python_modules/dagster/dagster/daemon/daemon.py

```diff
@@ -62,51 +62,56 @@ def run_daemon_loop(
         error_interval_seconds,
         until=None,
     ):
+        from dagit.cli import uploading_logging_thread
```

Comment:
this dependency seems undesirable - users might not have dagit installed on the daemon

Reply:
agreed - I can move this code to core and then import from there in both